packages/react/src/components/Tile/Tile.js

```diff
@@ -169,14 +168,14 @@ export class ClickableTile extends Component {
     );
 
     return (
-      <Link
+      <a
```

Comment:
I think this would be a breaking change by removing features previously supported by the Link component

Reply:
It seems like the only prop that causes a change is the `visited` prop, but we always want the text to be `#161616` anyways so not sure it's breaking. Is there other functionality we're providing via `Link` that is now unsupported?

Reply:
another functionality that would be broken is disabled links

Reply:
Seems like we've got an issue with that as-is. I'll create a separate issue to track that 

![disabled-tile](https://user-images.githubusercontent.com/11928039/112895731-7879fd00-90ab-11eb-94c6-6074e30c3a0f.gif)

Reply:
yeah the link itself should be disabled already so it looks like it would just be a visual issue. the root element probably does not need to be changed and the original referenced issue is just due to browser settings

Reply:
Agreed, this will prevent the incorrect color from being rendered if the link has been visited and retain the original element